test/Constraints/members.swift

```diff
@@ -737,7 +737,7 @@ func rdar66891544() {
   func foo<T>(_: T, defaultT: T? = nil) {}
   func foo<U>(_: U, defaultU: U? = nil) {}
 
-  foo(.bar) // expected-error {{cannot infer contextual base in reference to member 'bar'}}
+  foo(.bar) // expected-error {{type '_Copyable' has no member 'bar'}}
```

Comment:
Perhaps we should more generally do this for all marker protocols; a diagnostic like `type 'Sendable' has no member 'bar'" would be somewhat unexpected too for the same reason.

Reply:
Yeah, implicit conformances to marker protocols should not propagate (I am not sure about Sendable, haven’t reviewed any PRs for it).

Reply:
sure, I could make it general to exclude all marker protocols.... but you're allowed to add members to a marker protocol via extension, though you can't add static members:

```
extension Sendable {
  var member: Int { return 100 }
  typealias buddy = Int
}

func take<T: Sendable>(_ t: T) {}

func pass() {
  take(.member)    // error: instance member 'member' cannot be used on type 'Sendable'
  take(.nonmember) // error: type 'Sendable' has no member 'nonmember'
  take(.buddy)     // OK
}
```

so if I exclude marker protocols, all three calls turn into errors with "cannot infer contextual base in reference to member ...".

Since _Copyable is marked unavailable, you can't extend it, so nobody can add members to it.

Reply:
Then again, I don't know why anyone would want to add members to Sendable so I guess not inferring it is fine. They need to make it explicit.

Reply:
Since it’s not possible to add static members it shouldn’t be included because leading-dot syntax in generic context only works when first member in the chain is static.

Reply:
Just to clarify - it shouldn’t be included if it’s not stated explicitly, your example should still infer Sendable.